backend/src/database/migrations/V1684497543__init-default-segments.sql

```diff
@@ -0,0 +1,19 @@
+
+INSERT INTO segments
+    (id, name, "parentName", "grandparentName", slug, "parentSlug", "grandparentSlug", description, "sourceId", "sourceParentId", "tenantId")
+SELECT
+    -- this thing generates a valid uuidv4 without `uuid-ossp` extension: https://stackoverflow.com/a/21327318
+    uuid_in(overlay(overlay(md5(random()::text || ':' || random()::text) placing '4' from 13) placing to_hex(floor(random()*(11-8+1) + 8)::int)::text from 17)::cstring),
+    'Default', -- name
+    'Default', -- parentName
+    'Default', -- grandparentName
+    'default', -- slug
+    'default', -- parentSlug
+    'default', -- grandparentSlug
```

Comment:
`slug`, `parentSlug` and `grandparentSlug` has a unique index together. I think it's better to create the default segments of workspaces using the existing `tenant.url` field (it's also slug-like and unique)